extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -4,6 +4,7 @@ class PortableInfoboxRenderService extends WikiaService {
 	const LOGGER_LABEL = 'portable-infobox-render-not-supported-type';
 	const DESKTOP_THUMBNAIL_WIDTH = 270;
 	const MOBILE_THUMBNAIL_WIDTH = 360;
+	const MIN_BIG_IMAGE_WIDTH = 300;
```

Comment:
this const is for minimal supported image width for hero image in infobox, please consider different naming, for example: `MIN_SUPPORTED_HERO_IMG_WIDTH` or `MINIMAL_HERO_IMG_WIDTH`